src/Module.php

```diff
@@ -155,7 +155,13 @@ final class Module extends \luya\admin\base\Module implements CoreModuleInterfac
      * @since 1.2.0
      */
     public $userIdleTimeout = 1800;
-    
+
+    /**
+     * @var string The component/connection name from application.
+     * @since 2.0.0
+     */
+    public $proxyDbConnection = 'db';
```

Comment:
I just made a quick search for `proxyDbConnection` - it seems this property is not used - am i wrong? sorry if so.

Reply:
Oh thank you, my mistake while refactoring `ProxyController`